### PR TITLE
Unregister metrics when Application is unloaded. Fixes #137. (17.0.0.3_RC) 

### LINKED ISSUE #137: Unregister Metrics when Applications are unloaded

### PR DESCRIPTION
**Pull request by raymondlam**

### CODE REVIEW

**Comment by raymondlam** [comment]
#build

File: dev/com.ibm.ws.microprofile.metrics/src/com/ibm/ws/microprofile/metrics/impl/MetricRegistryImpl.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
-* Copyright (c) 2017 IBM Corporation and others.
+H* Copyright (c) 2017 IBM Corporation and others.
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics/src/com/ibm/ws/microprofile/metrics/impl/MetricRegistryImpl.java]: An extra 'H'

**Comment by raymondlam** [comment]
Fixed extra H in copyright header

**Comment by raymondlam** [comment]
Build at
https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_qAvyUJ75Eeew8ISV5CV8GQ&tab=com.ibm.team.build.web.ui.internal.editors.result.overview.OverviewPage

This was tested manually using the Microprofile metrics TCK and a sample demo application

**Comment by raymondlam** [comment]
Test failure in WebModuleMBeanTest is unrelated to changes.

@OpenLiberty/delivery-approvers Can you please merge?

**Comment by raymondlam** [comment]
Fixes #137

**Pull request merged**